ws-nextjs-app/pages/[service]/new_live/[id]/Posts/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import path from 'ramda/src/path';
+import pathOr from 'ramda/src/pathOr';
+import { OptimoBlock } from '#models/types/optimo';
+import headings from '#app/legacy/containers/Headings';
+import Blocks from '#app/legacy/containers/Blocks';
+import paragraph from '#app/legacy/containers/Paragraph';
+import Text from '#app/components/Text';
+import { Post, StreamResponse } from './post.d';
+
+// temporary solution to render LI/ OL blocks.
+const unorderedList = ({ blocks }: { blocks: OptimoBlock[] }) => {
+  const listItems: (string | null)[] = blocks
+    .map(item =>
+      pathOr<string | null>(
+        null,
+        ['model', 'blocks', 0, 'model', 'text'],
+        item,
+      ),
+    )
+    .filter(text => typeof text === 'string');
+
+  return (
+    <Text>
+      <ul>
+        {listItems.map(item => (
+          <li>{item}</li>
+        ))}
+      </ul>
+    </Text>
+  );
+};
+
+const PostHeadings = ({ headerBlocks }: { headerBlocks: OptimoBlock[] }) => {
+  const componentsToRender = {
+    headline: headings,
+    subheadline: headings,
+  };
+
+  return (
+    <Blocks blocks={headerBlocks} componentsToRender={componentsToRender} />
+  );
+};
+
+const PostContent = ({ contentBlocks }: { contentBlocks: OptimoBlock[] }) => {
+  const componentsToRender = {
+    paragraph,
+    unorderedList,
+    orderedList: unorderedList,
+  };
+
+  return (
+    <div>
+      <Blocks blocks={contentBlocks} componentsToRender={componentsToRender} />
+    </div>
+  );
+};
+
+const PostItem = ({ postItem }: { postItem: Post }) => {
+  const headerBlocks = pathOr<OptimoBlock[]>(
+    [],
+    ['header', 'model', 'blocks'],
+    postItem,
+  );
+
+  const contentBlocks = pathOr<OptimoBlock[]>(
+    [],
+    ['content', 'model', 'blocks'],
+    postItem,
+  );
+
+  return (
+    <div>
+      <PostHeadings headerBlocks={headerBlocks} />
+      <PostContent contentBlocks={contentBlocks} />
+      <hr />
+    </div>
+  );
+};
+
+const PostsList = ({ postData }: { postData: StreamResponse }) => {
+  const postResults = path<[]>(['data', 'results'], postData);
```

Comment:
Can we avoid using ramda path, and just use optional chaining + destructuring instead? 
```suggestion
  const { results: postResults } = postData?.data;
```

Reply:
Yes indeedy, good shout!